01-node-tutorial/answers/writeWithPromisesAwait.js

```diff
@@ -0,0 +1,41 @@
+//getting the built in file system module
+const {writeFile, readFile } = require("fs").promises;
+
+//3 lines
+const content = `This is line 1, 
+This is line dos,
+Lastly line 3.`;
+
+//async writer function
+async function writer(){
+
+    try{
```

Comment:
More of a nit picky thing, but you should get a code formatter, like prettier, to run to automatically format your code and make it more readable in future assignments. That way you never have to worry about spaces and its makes it more readable for everyone.

Reply:
I have prettier installed so I am not sure why it turned out that way.